src/AppKit/Enums.cs

```diff
@@ -138,9 +139,11 @@ public enum NSComposite : nuint_compat_int {
 
 	[Native]
 	public enum NSBackingStore : nuint_compat_int {
-		[Availability (Introduced = Platform.Mac_10_0, Deprecated = Platform.Mac_10_13, Message = "Use 'Buffered' instead.")]
+		[Mac (10, 0)]
```

Comment:
Similar to the iOS < 6.0, we should be able to drop Mac tags < 7.0

Reply:
Mac tags < 10.7 (: